go/worker/keymanager/p2p/protocol.go

```diff
@@ -0,0 +1,31 @@
+package p2p
+
+import (
+	"time"
+
+	"github.com/oasisprotocol/oasis-core/go/common/version"
+)
+
+// KeyManagerProtocolID is a unique protocol identifier for the keymanager protocol.
+const KeyManagerProtocolID = "keymanager"
+
+// KeyManagerProtocolVersion is the supported version of the keymanager protocol.
+var KeyManagerProtocolVersion = version.Version{Major: 1, Minor: 0, Patch: 0}
+
+// Constants related to the GetDiff method.
+const (
+	MethodCallEnclave          = "CallEnclave"
+	MaxCallEnclaveResponseTime = 5 * time.Second
```

Comment:
Yeah we could do that, but the server handler can already do it by setting a context with a timeout (this is not possible on the client side as this would only configure a global timeout since client calls may go out to multiple peers).